Base/usr/share/man/man1/cal.md

```diff
@@ -0,0 +1,77 @@
+## Name
+
+cal - Display a calendar
+
+## Synopsis
+
+```**sh
+$ cal [[[day] month] year]
+```
+
+## Description
+
+This program displays a simple calendar. If no arguments are specified, the current month is displayed with the current day highlighted.
```

Comment:
This sentence appears very similar to the one from the [util-linux man page](https://github.com/karelzak/util-linux/blob/master/misc-utils/cal.1), which is under 4-clause BSD. I guess there's only so many ways one can describe what this program does, but maybe we should make it intentionally more different just to be sure?